files/ohpc_install.yml

```diff
@@ -0,0 +1,226 @@
+ - job:
+         name: ohpc_install
+         description: "This is the job to install OpenHPC on a cluster"
+         project-type: freestyle
+         block-downstream: false
+         concurrent: true
+         properties:
+                 - authorization:
+                         hpc-sig-admin:
+                                 - credentials-create
+                                 - credentials-delete
+                                 - credentials-manage-domains
+                                 - credentials-update
+                                 - credentials-view
+                                 - job-build
+                                 - job-cancel
+                                 - job-configure
+                                 - job-delete
+                                 - job-discover
+                                 - job-move
+                                 - job-read
+                                 - job-status
+                                 - job-workspace
+                                 - ownership-jobs
+                                 - run-delete
+                                 - run-update
+                                 - scm-tag
+                         hpc-sig-devel:
+                                 - job-build
+                                 - job-read
+         parameters:
+                 - node:
+                         name: node
+                         default: ''
+                         description: 'Node on which to execute this job'
+                 - choice:
+                         name: cluster
+                         choices:
+                                 - d05
+                                 - qdc
+                         default: 'qdc'
+                         description: 'The MrP type of the machine to be provisioned'
+                 - choice:
+                         name: method
+                         choices:
+                                 - stateful
+                                 - stateless
+                         default: 'stateful'
+                         description: 'The type of OHPC install to do'
+                 - choice:
+                         name: ohpc_recipe_repo
+                         choices:
+                                 - takekato
```

Comment:
if anything, this should be a text field.
I'd prefer same as others, a fixed upstream repo with branch as an option.
We can have it pointing to yours and you make sure kato's is up-to-date in a branch.